CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
-# 1.9.0 (2023-07-11)
+# Changelog
+
+## [1.9.0] - 2023-07-11
+[1.9.0]: https://github.com/crystal-lang/crystal/releases/1.9.0
```

Comment:
Out of curiosity why not just link it directly?

```suggestion
## [1.9.0](https://github.com/crystal-lang/crystal/releases/1.9.0) - 2023-07-11
```

Typically the reference-style link targets are all at the end of the document, like a footer, so I was surprised to see it immediately after where it's used.